getting_started/first_app.md

```diff
@@ -365,16 +365,9 @@ We integrated our [CLI](https://github.com/adobe/aio-cli) with an open source pr
 - Auto-invoking of actions on code changes
 - Or running any shell command such as a curl request on code changes
 
-To debug your application, you will need to install some additional dependencies if you only set up the required tools, see  `Optional tools` section in  [Setting up Your Environment](setup.md). When you have the dependencies installed, make sure that local dev environment is running with `aio app run`, `--local` an option. 
+If the local development is run without the `--local` option, the action you are debugging is running directly on [Adobe I/O Runtime](https://www.adobe.io/apis/experienceplatform/runtime.html). When the `--local` option is used, the actions are debugged in the standalone OpenWhisk instance running locally. Learn more about what you can do in [debug mode](https://www.npmjs.com/package/@adobe/wskdebug). 
```

Comment:
The linked package is deprecated since wskdebug has moved to OW. See https://www.npmjs.com/package/@openwhisk/wskdebug

Also should we mention that in both cases (whether --local is used or not), Docker is required on the local machine? Ultimately, even remote actions get proxied to the local Docker container, but the locally mounted acitons inherit from the live context of the deployed ones. See https://github.com/apache/openwhisk-wskdebug#how-it-works for reference.

Reply:
thanks @meryllblanchet , updated accordingly 👍